script/DeployMockToken.s.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity ^0.8.17;
+
+import "forge-std/Script.sol";
+import { ERC20Mock } from "openzeppelin-contracts/mocks/ERC20Mock.sol";
+
+contract DeployStreamAndFactory is Script {
```

Comment:
@eladmallel not sure if this file is used, but the contract name doesn't match the content

Reply:
you're right, will fix. it is used when testing locally, need a payment token

Reply:
fixed